scripts/helpers/cookies.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Cookies helper
+ * Helper to set and unset cookies.
```

Comment:
Perhaps listing the available helper methods for time (`setOneDay` etc.) would be useful.